### ISSUE TITLE
shortcuts

### ISSUE THREAD

**Issue by Ganaboina** [Ganaboina/shortcuts#1]
howd1 How was your day so far?
alt I have an alternate resolution for this issue.
time2 Sure, please take your own time.
gth Good to hear that from you.
howd2	How are you doing today?
howd3	I'm doing great, thank you for asking.
see1	It's nice to have you on this chat.
ty	Thank you.
y1	You're welcome.
take1	Take care of yourselves!!!
mp	My pleasure.
aw	Awesome!
jtc	Just to confirm,
gr	Great!
al	Alright!
fn	Fantastic!
pr	Perfect!
ex	Excellent!
won	Wonderful!
np1	No problem.
nw1	No worries.
tf1	That's fine.
to1	That's okay.
ta1	That's alright.
tr1	That's right.
trc1	You are absolutely correct.
tyo	Thank you for being online.
pat	I appreciate your patience.
rat	I really appreciate that.
tyc	Thank you for confirming.
tyi	Thank you for the information.
tys	Thank you for being supportive.
tyu	Thank you for understanding.
tyr	Thank you for responding.
typ	Thank you for the permission.
icu	I completely understand your concern.
icu2	I understand.
icu3	I can understand. 
ihope	I hope we are connected.
ihopeu	I hope you understand.
sry	I am sorry to know that.
srh	I am sorry to hear that.
take2	Please take good care of yourself!!!
h1	Hello!
r1	How are you doing?
ght	Good to hear that.
gla	Sure, I'll be glad to help you with that.
glar	I'll be happy to assist you with your request.
glai	I'll be happy to help you with that information.
gknow	I am glad to know that.
ghear	I am glad to hear that.
letme	Let me check that for you.

letmen	Let me know once you are done.
letmec	Let me check what best I can do for you.
ill	I'll be happy to stay online with you.
iam	I am working on your request.
pgo	Yes, please go ahead.
sgo	Sure, please go ahead.
allo	Please allow me a moment while I work on it.
iso	That would help me isolate the issue. 
ge	Good evening.
gn	Good night.
gm	Good morning.
gld	Sure, I'll be glad to help  you with that issue right away.
iss1	Please let me know if you have any issues.
ir3	Was I able to resolve your concerns up to your satisfaction today? 
st1	I'll standby for you.
bye	Take care and Be safe!!!
bye1	Have a Wonderful night!
bye2	Have a Wonderful day!
tl1	Thank you for letting me know.
ord1	Please make a note of this order number :
sat1	Your satisfaction is what I strive for.
tysm	Thank you so much!
check1	Please check if it works fine now.
concern1	I completely understand your concern.
val1	Anytime for our valued customers.

hi	How are you doing today?
g1	That is good to hear from you.
g2	I am fine. Thank you very much for asking.
e1	I request you to please elaborate your issue so that I can assist you in the best possible way.
b1	I am sorry to hear that.
p1	I see that you are facing issue with the . Am I correct?
p2	I see that you want to . Am I correct?
do1	Please allow me a minute of your valuable time while I do that for you.
do2	Please let me know once you are done.
tnq	Thank you for being online. I appreciate your patience.
howmay	How may I assist you today?
isthat	Is that okay for you?
set1	You are all set now.
det1	I'll explain that for you. 
det2	I'll explain that in detailed for you.
put1	I am putting all my efforts and knowledge in this to fix the issue for you.
fine2	In this case, I will go ahead and provide you some troubleshooting steps.

myac	Good to know information. You can always visit our website https://myservices.timewarnercable.com/ and sign into your 

account for setting digital phone features, checking order status, checking for outages, billing, making payments, resetting Spectrum 

equipment and check tech visit updates.
ple1	Please allow me 2 minutes while I access your account.
ple2	I am sorry it is taking longer than expected. I am still working on your issue. Please allow me another 2-3 minutes.
clo1	We value you as a customer and please feel free to contact us any time if you have any additional queries or concerns as we are 

here to assist you 24 hours a day, 7 days a week.

is1	Is there anything else that I may help you with today?
is2	Have I resolved all your concerns to your satisfaction today?
is3	Have I answered all your questions to your satisfaction today?
is4	Are you satisfied with my services today?

op0	Hello. This is Rakesh. I see you've already been chatting. Please give me a moment so I can read the previous chat and pick up 

where you left off.
op1	Thank you for contacting us . My name is Rakesh. I'd be happy to assist you.
op2	Thank you for contacting us. My name is Rakesh. Please give me a moment while I review the information you've provided. 
c0	I hope that I have answered all of your questions today. It was nice chatting with you. Thank you for contacting us. Have a 

great day!

h1	Hello!
r1	How are you doing?
ght	Good to hear that.
ass1	Please be assured, I'll take complete ownership to resolve this issue for you in this chat.

let1	Let me check that for you.
quick1	Please allow me a quick minute.	
gtk	Please save above link for your future reference.
home1	Customer satisfaction is our main priority. Have I answered all your concerns for your satisfaction today?
home2	Customer satisfaction is our main priority. Have I resolved all your issues for your satisfaction today?
ght1	I'm doing great. Thanks for asking.

para	I understand you're experiencing an issue with XXXX. Am I correct? 

ship1	It seems that there is some issue with the cable box you are using. In this case I will ship a new cable box to your place for free 

of cost.
ship2	I have shipped the best available cable box in the market for you.
ship3	You will be receiving the cable box in 1-2 days to your place from now.

note1	I will document a detailed note on your account and report it to the concerned department regarding the issue you are facing 

and make sure your issue is resolved at the earliest and make sure you will not face this issue again in future.

note2	I am now escalating your concerns to the relevant department to handle this issue and resolve at the earliest for you.

note3	I will also involve my technical team to resolve your issue at the earliest as one of the loyal customer of TWC is facing the 

issue.

note4	I now reported your issue to our Technical Expertise team and they are now in the process of resolving your issue at the earliest 

for you.	

trans0	As it is a Technical Chat Repair for Texas, I will have to transfer this chat to the concerned department who will take care of 

this for you.
trans1	At this point of time I will transfer this chat to our advanced technical team who can help you in resolving the issue right away 

for you.
trans2	Please do not disconnect the chat session, this might take a minute or two. So, please stay connected while I refer this chat to 

the concerned team.

senti1	My foremost priority over this chat is to make each and every TWC customer happy with my assistance.
senti2	Your satisfaction is very important for me.
senti3	Trust me I am putting all my efforts to fix the issue at the earliest for you.
senti4	I am really sorry from my end as one of the loyal customer of TWC is facing the issue.

----------------------------
Internet Troubleshooting
----------------------------

int1	Let us follow all possible troubleshooting steps to resolve this issue.
int2	I request you to flip the ends of the co-axial cable from the wall outlet to the modem and re plug it back securely.
int3	Let me go ahead and send some advanced re configuration codes to your modem now. This will help in removing the network 

congestion and accumulation.
int4	Good to know information for you future reference. You can always visit our website for issues related to Internet services, 

please visit our self help article for future reference: http://www.timewarnercable.com/en/support/internet/step-by-step/internet-

troubleshooting.html

int5	I've successfully sent advance re configuration codes to your modem.
int6	I will now run few diagnostics from my end on your service line remotely to check if there is an problem with the Internet.
int7	Please allow me a minute while I diagnose signal's strength on your modem remotely.
int8	I've verified the signal levels from my end and they are pretty good now.
int9	The good news is that now I do not see any packet loss and latency issues and the signals are boosting up to a great extent.
int10	I have made all the necessary changes on your account from my end now so that you will not face any such issues and stay 

happy with your services.
int11	I now see everything is perfectly setup on your account.
int12	I assure you that you will not face any such issues in future.
int13	I am taking complete ownership of this.
int14	All you need to do is once after the chat session is done reboot the modem once for the changes that I have made to take effect.
int15	Once the reboot is done you will enjoy the best of services.
int16	This will fix the issue permanently for you.
int17	I have successfully isolated the issue and provided you with a permanent solution.

----------------------------
Frustration
----------------------------
frus1	Please be assured I always have the best of intentions when serving my customers. Trust me I will put all my efforts to fix the 

issue at the earliest for you.
frus2	I do not want to see you upset anymore, I will do all the things which are in my power to fix the issue.
frus3	When you do not receive the level of attention you deserve, our entire team is dedicated to correcting the problem, and I will 

make sure it does not happen to you again.
----------------------------
Verification
----------------------------
ver1	For your security and to ensure we protect your privacy, please provide me with the last four of SSN.
ver2	For your security and to ensure we protect your privacy, please provide me with name, address, Customer Code, home Phone 

or account contact number.

per1	In this case I have two permanent solutions to fix the issue for you.
per2	Either you could visit any nearest TWC Store and swap the box with an enhanced one free of cost or I can schedule our expert 

technician to visit your place and fix it for you. I am taking the complete ownership of it.
per3	I am also documenting a detailed notes on your account regarding this.
per4	Once you walk into the local store you can ask the local store staff to refer the notes on your account regarding this so that they 

will provide you with the best possible device available in the market. 

ref1	While you are waiting, I will provide you a link for your future reference.
ref2	In the meantime, I will provide you with a helpful link for your future reference.
-------------------------------
TV Troubleshooting
-------------------------------
tv1	Let us follow all possible troubleshooting steps to resolve this issue.
tv2	I request you to flip the ends of co-axial cable from the wall outlet to the cable box and re plug it back securely.
tv3	Let me go ahead and send some re configuration codes to your cable box now. This will help in removing the network 

congestion and accumulation.
tv4	Good to know information for your future reference for issues related to cable TV issues, please visit our self help article for 

future reference: https://www.timewarnercable.com/en/support/tv/step-by-step/tv-picture-quality.html

tv5	I've successfully sent advance re configuration codes to your cable box.
tv6	I request you to reboot the cable box by un plugging the cable from the power cord and re plugging it back after 30 seconds.
tv7	We will wait for the cable box to complete its reboot and check if everything works fine.
tv8	Generally the cable box takes 3-4 minutes for the reboot process to complete.
tv9	During the reboot process, several things will display on the front panel of the box: BOOT, APP, AIT and OCAP. Finally, the 

box will display a count down from L-13 to L-1.
tv10	Once this is done we will check if everything works fine.

phone0	For issues related to home phone services, please visit our self help article for future reference: 

http://www.timewarnercable.com/en/support/phone/step-by-step/phone-troubleshooting.html for step-by-step instructions.
email0	If you ever need help with the TWC email, please visit http://www.timewarnercable.com/en/support/internet/step-by-

step/twc-mail.html, With the help of the above link, you can find details on how to use our webmail services, create email sub accounts, 

along with the server settings for different client emails on your phone or computer device such as Microsoft Outlook, Windows Live 

Mail, Macintosh OSX mail, etc.

app1	I really appreciate all your efforts in troubleshooting the issue.
app2	I really appreciate your time and efforts.

fix0	I really want to thank you for being supportive and for co­operating with me while I had a chance to fix this issue.
fix1	I appreciate all your efforts. I would not be able to fix this issue without your co-operation.
fix2	I really thank you for giving me the opportunity to assist you.	
fix3	I thank you for your co-operation throughout the chat.
fix4	I appreciate you for being very supportive.
fix5	It was a pleasure chatting with you, you are a valued customer for me.
fix6	You have been a wonderful customer to assist with!
bis1	I can understand how upsetting it must be to not have cable at home as I've been through such situations as well. At the end of 

the day I am a customer as well.
bis2	I assure you that you will not face this issue again. Trust me all my efforts and hard work will be fruitful for you.
bis3	Take my words you will not face this issue again.
credit1	I will process you credits for you right away as you have faced the issue.
credit2	The credits will be adjusted in your next month bill.

email00	If you ever face any issues with the email services in future.• You can also view our FAQs on Email by visiting 

http://www.timewarnercable.com/en/support/all-faqs.html and picking a topic as "Internet" from the "Select a Topic" dropdown box. • 

Pick a sub topic as Email, then select the question that you want an answer to, from the list of questions listed on the right hand side.• If 

you aren't able to find the answer in the list of FAQs, please contact us for further assistance. 
wifi00	If you ever face any issues with the wifi services in fututre.• You can also view our FAQs on Home WiFi by visiting 

http://www.timewarnercable.com/en/support/all-faqs.html and picking a topic as "Internet" from the "Select a Topic" dropdown box. • 

Pick a sub topic as Home WiFi, then select the question that you want an answer to, from the list of questions listed on the right hand 

side.• If you aren't able to find the answer in the list of FAQs, please contact us for further assistance. 
int00	If you face any issues with your Internet services in future I request you to once check if all the connections are hand tight and 

once reboot the modem if you still face any issues you can always feel free to contact us so that we will provide you with the best 

resolution.
phone00	If you face any issues with your phone services in future I request you to once check if all the connections are hand tight and 

once reboot the modem if you still face any issues you can always feel free to contact us so that we will provide you with the best 

resolution.
tv00	If you face any issues with your TV services in future I request you to once check if all the connections are hand tight and once 

reboot the cable box if you still face any issues you can always feel free to contact us so that we will provide you with the best resolution.

compli1	It's such a compliment which helps me to serve you better.
no1	I have not heard from you in a while and your chat window shows you've been idle, are you still with me?
no2	I am afraid, this chat might be disconnected as there is no response from you.
no3	I apologize, however due to the lack of response the chat will get disconnects automatically. Please feel free to contact us any 

time with additional questions or concerns. Thank you for contacting.

c0	I hope that I have answered all of your questions today. It was nice chatting with you. Thank you for contacting us. Have a 

great day!
c2	Please click on End Conversation tab to close the chat session. 

link1	I have placed a clickable link to reconnect us. Clicking on the link will allow you to get connected back to me, then we may 

continue. If for some reason I am unavailable once you attempt to reconnect, a peer of mine will be able to pick up where we leave off.
v1	Could you please provide me your first and last name, along with the telephone number listed on the account?

fine1	Is that fine with you?
hap1	Have a great Holiday season! 
pull1	Please help me with the phone number or account number to pull up your account details. 
hap2	Merry Christmas to you and your family! 
about1	If you are unsure, please open your email as you normally would, click on the Help menu (at the top beside File, Edit View, etc) 

and click on About. In there you will find the name of the program and the version. Please let me know what they are.
apo0	I apologize once again and thank you for being patient.
apo1	I sincerely apologize for the inconvenience caused to you and I will try my best to resolve it for you.
apo2	I do apologize for this.
apo3	I sincerely apologize for the inconvenience caused to you and I will be glad to help you with the issue.
are1	Are you able to login with above Password?
are2	Are you done with above steps?
yes1	Yes, please go ahead with your query. 
typo1	Sorry for the typo. Please ignore the above statement.
typo2	I apologize for the sentence that just reached you. That was typed in error. Please ignore it.
voice1	I understand there is a heavy call flow and our voice support agents are helping our valuable customers like you.
prof1	Please be advised that this is a professional chat. I am sorry to inform you that, I will be forced to disconnect the chat if you are 

using un parliamentary words.

know1	Let me know once you are done with the above steps.
know2	After performing the above steps. Check for the issue and let me know the result so that we can perform other troubleshooting 

steps to fix the issue.
know3	Let me know once you are logged in.
know4	Let me know once you are logged in with Temporary Password

hold1	Please allow me 2 minutes while I check the information for you.
hold2	Please allow me 2 minutes while I work on the issue. 
hold3	Please allow me 2 minutes of your precious time while I check that for you.
hold4	Thank you for being on hold, I appreciate your patience.
hold5	I am sorry this is taking more time than usual, please allow me more two minutes.
hold6	It is taking much longer than it is expected. Please bear with me.

help1	I understand your concern, I will help you to resolve the issue, to the best of your satisfaction. 
help2	I will definitely help you with the issue.
help3	Sure, I am glad to assist you with this regard.
help4	I will certainly help you in resolving your issue.
help5	I apologize for the inconvenience caused. I would be glad to assist you regarding this issue.
help6	I'm sorry to hear that you are facing an issue. I'll be happy to resolve that for you.
help7	I'm sorry to hear that you are facing an issue. I'll be happy to help you with that.

free0	You need to delete the unwanted emails from inbox, sent, drafts, personal folders and lastly from trash folder of webmail to free 

up the space on webmail. 
error1	May I know the exact error message displayed when you try to send and receive the email messages?
error2	In order to assist you better, please provide me the exact error message you received on the screen.

glad1	I'm glad I was able to help you with the issue.
it1	It was my pleasure assisting you today. 
it2	It was nice chatting with you.
und3	We don't like to see our customers upset and inconvenienced. We always strive to create a positive customer experience.
und4	The same thing happened to me only recently, let me try and see what I can do to rectify the problem.
sorry1	I am sorry to know that.

hit1	I will go ahead and send data string codes to your modem which will clear the modem's internal memory and Network Traffic 

that may cause internal problem, this will improve your Internet connection.
hit2	I've successfully sent data string codes to your modem to refresh the services. 
hit3	I will go ahead and send data string codes to your cable box which will clear the box's internal memory and Network Traffic 

that may cause internal problem.
hit4	I've successfully sent data string codes to your cable box to refresh the services. 

ver1	For security reasons, may I have the PIN on your account?
ver2	For security reasons, may I have the last four digits of SSN please?
ver3	For security reasons, may I have the account holder's name, Address, Customer code and Phone number please?
tr1	You've reached Texas technical support. Let me transfer the chat to dedicated department for further assistance.
tr2	You've reached New York technical support. Let me transfer the chat to dedicated department for further assistance.?
 
reset4	I suggest you please reset the email password at : https://selfcare.rr.com

reboot1	Please reboot your modem by unplugging power cord from the electrical outlet, wait for 30 seconds and plug it back in.
reboot2	Please reboot your cable box by unplugging power cord from the electrical outlet, wait for 30 seconds and plug it back in.

code1	A Customer Code is a four digit number located at the top right hand side on your bill, under the account number.
cust1	Please provide me with your Customer Code. A Customer Code is a four digit number located at the top right of your bill, under 

the account number.

pro1	How long have you been experiencing the issue?
pro2	Were there any changes made from the time it was working fine?
pro3	Is the phone line connected directly to modem or the wall jack?
pro4	Is the issue with wired connection or wireless?
sec1	If you are thinking about the chat security, I would like to inform you that you are currently in a secure TWC LiveChat session, 

in direct contact with a TWC LiveChat Technical Support Representative and this is a secure TWC site with 128 bit encryption which 

does not allow any third party to access the information of our Chat conversation.
act1	Please activate the modem at http://elb02.activatemymodem.com/landing
res1	I've checked the information for you.
res2	I see that the signals are good from our end.
sales1	855-892-1002
cncl	888-892-2253
cncl1	800-222-5355 
promo	1-888-892-2253 
cblcard	1866-532-2598 Hours 8 AM to 11 PM
mail1	You can mail or print this chat transcript by clicking on the envelope or printer icon on the top of this window.
cal1 Please provide me your preferred contact number for me to schedule the call back.
cal2 I have successfully scheduled a call back and you will be informed as soon as the maintenance is done.

wi1	Please open a new tab or browser and type in the IP 192.168.0.1 in the address bar (URL bar) and press enter.
wi2	This will take you to the router login page.
wi3	I request you to login using Username: admin and Password: password
wi4	Please go to the Wireless tab >> Basic tab.
wi5	You can find the WIFI network name in the SSID field and the password in the Pre-Shared Key field.
wi6	You can also change the password or the WIFI network name from the same page by changing the SSID and the Pre-Shared 

key and clicking on the Apply.
wi7	Would you like me to access router page remotely from my end?
hold0	Please allow me 2 minutes of your time to run a diagnostic test on the modem remotely to isolate the issue. 
isol	I have successfully isolated he issue from my end now. I see that the firmware and boot version on your modem is outdated and 

the channel wasn't set up properly on modem hence you're experiencing the issue.
chang	I am making the required changes now and increasing the bandwidth on the modem. I've made the required changes in the 

modem's settings and have updated the firmware and boot version on it.

resp1	It is my responsibility to put a smile on each and every customer's face by resolving their issue as quickly as possible.
resp2	I wish I have customer's like you in future for support. You were really good through out the chat today.
fore1	My foremost priority over this chat is to see the customer happy with my assistance
smile1	It is my responsibility to put a smile to each TWC customer with my service
resp3	It is my responsibility. I will try my best to fix this issue by the end of this chat.
supp1	Thank you for your support and we are pleased to have customers like you.
limit1	I would have loved to sort this out for you right away. I trust you'll understand my limitation, that I am a member of Online 

Customer Services Team and have limited access to certain tools and applications due to which I am unable to process your request from 

my end.

mac1	May I have the MAC address (C mac) of your device? It is located at the rear of the device on a sticker.
own1	As a TWC representative I will take the ownership and personally follow up this for you.
own2	You have reached the right person and I will be glad to take care of it for you.

how1	How are you doing,  ?
gr1	Great to hear that from you. 
fab1	I am doing fabulous, thank you for asking.
faq1	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic from the "Select a Topic" dropdown box. 
faq2	Pick a sub topic relevant to your question, then select the question that you want an answer to, from the list of questions listed 

on the right hand side.
faq3	If you aren't able to find the answer in the list of FAQs, please contact us for further assistance.
must1	For your convenience, You can also check for known outages or check the status of an outage in your area by logging in to the 

My TWC App (https://mytwc.mpsvcs.com/) or My Account (https://myservices.timewarnercable.com). 
must2	You can also upgrade your services, manage your phone and its features, set digital phone features, check your order status, 

check for outages, view your billing, make a payment, reset your TWC equipment and check tech visit updates by visiting our website 

https://myservices.timewarnercable.com/	
fut1	The above link is for your future reference.
self1	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "Internet" from the "Select a Topic" dropdown box. Pick a sub topic as Internet Plans, then select the question that you 

want an answer to, from the list of questions listed on the right hand side. 
self2	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "TV" from the "Select a Topic" dropdown box. Pick a sub topic relevant to your question, then select the question that 

you want an answer to, from the list of questions listed on the right hand side. 
self3	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "Phone" from the "Select a Topic" dropdown box. Pick a sub topic relevant to your question, then select the question 

that you want an answer to, from the list of questions listed on the right hand side. 

boot1	During the boot process, several things will display on the front panel of the box: BOOT, APP, AIT and OCAP. Finally, the box 

will display a count down from L-8 to L-1.

online1	I will be more than happy to stay online for you. Please let me know once you're done.
done1	I've completed all the troubleshooting steps I'm able to assist you with. I would like to schedule a technician visit so the 

connections and devices can be physically checked and the issue isolated.

speeds1	I see that there is no outage affected in your area. The signals are good from our end. The good news is that I can fix this issue 

now. I just have to send the signals to boost signal levels and change the channel frequencies on routers page. I am going to set up the best 

channel frequency on your modem. This channel frequency will give you high speeds with consistent connectivity. Can I set up channel 

frequency for you? 
speeds2	Please allow me a minute to set the best channel frequency for you. I have successfully set the best channel frequency on your 

modem now and have also sent the signals. To update this on your end, let me reboot the modem on the network so we will be all set. This 

will disconnect the connection for a moment and you will be online again with a minute or two.
speeds3	Rest assured, everything has been taken care of. I will ensure you won't have to get back to us for the same issue again. The 

issue will be fixed permanently. I will reboot the modem after we end the chat session.
speeds4	I request you to check the speeds in next 10 minutes and everything will be in place there after. I'm glad that I was able to 

resolve your issue. Is there any thing else I can help you with?

tss1	Let's follow some troubleshooting steps to resolve this issue for you right away.
okay1	Are you okay with that? 
okay2	Does that work for you?
okay3	Is that okay for you?
o1	I see that there is no outage affected in your area. The signals are good from our end.
reset1	Let us try to factory reset the modem, I request you to press the reset button which is located at the rare end of the modem. 

Please press it for 10 seconds and then release. All the settings will be restored to it's default.
upd1	Let me quickly go through your account and check for the updates.
2min	Please allow me 2 minutes of your precious time.
care1	I will ensure everything is taken care before we end this chat session.

status1	Please keep me updated with the status. 
status2	May I know the status please?
status3	Thank you for updating me the status.
goal1	My goal was to provide you with an excellent customer service. Did I do that for you today? 
assure1	I assure you that you will not experience any issues going forward. I will ensure you won't have to get back to us for the same 

issue again. The issue will be fixed permanently. 

keep1	Please keep me update with the status of reboot.
smile1	I assure you that, at the end of the chat I will make you smile. 
over1	We have done it. Thank you for your support. 
fix1	Should I close our conversation as issue resolved, then?
g1	Great, I am very happy to know that.
ef1	I really appreciate your efforts in doing that.
ef2	I really appreciate all of your efforts in troubleshooting the issue.
und1	I do understand that and trust me if I were you I would have feel the same.
und2	I know what you're trying to say, I will do my best to help you out.
und3	The same thing happened to me only recently, let me try and see what I can do to rectify the problem.
und4	I completely understand the situation. I agree with you in this regard.
und5	If I were in your place, I would feel the same if I come across such situation.

nice1	It was pleasure assisting you today. Thank you for being so polite and patient through out the chat.
nice2	I want to make sure you have the best experience possible today with our chat service. 

longer1	It will not be longer because we know our customers wouldn't be happy with it.
pre1	May I have your preferred contact number to update you?
lot1	We have lot of customer's who are impacted, but it is for the improvement of services only.Hope you understand. 

name1	May I have your name please?
rap3 May I know your favourite weekend spot?
rap2	May I know your favourite holiday place? 
rap1	May I know your hobbies? 

ass1	Please be assured, I'll take complete ownership to resolve this issue for you in this chat.
gtk	Please save above link for your future reference.
home1	Customer satisfaction is our main priority. Have I answered all your concerns for your satisfaction today?
home2	Customer satisfaction is our main priority. Have I resolved all your issues for your satisfaction today?
ght1	I'm doing great. Thanks for asking.
 

no1	I have not heard from you in a while and your chat window shows you've been idle, are you still with me?
no2	I am afraid, this chat might be disconnected as there is no response from you.
no3	I apologize, however due to the lack of response the chat will get disconnects automatically. Please feel free to contact us any 

time with additional questions or concerns. Thank you for contacting us.

v1	Could you please provide me your first and last name, along with the telephone number listed on the account?
t1	Thank you for the information.
t2	Perfect. Thank you for verifying.
t3	Perfect. Thank you for confirming.
fine1	Is that fine with you?
hap1	Have a great Holiday season! 
pull1	Please help me with the phone number or account number to pull up your account details. 
hap2	Merry Christmas to you and your family! 
about1	If you are unsure, please open your email as you normally would, click on the Help menu (at the top beside File, Edit View, etc) 

and click on About. In there you will find the name of the program and the version. Please let me know what they are.
apo0	I apologize once again and thank you for being patient.
apo1	I sincerely apologize for the inconvenience caused to you and I will try my best to resolve it for you.
apo2	I do apologize for this.
apo3	I sincerely apologize for the inconvenience caused to you and I will be glad to help you with the issue.
are1	Are you able to login with above Password?
are2	Are you done with above steps? 
typo1	Sorry for the typo. Please ignore the above statement.
typo2	I apologize for the sentence that just reached you. That was typed in error. Please ignore it.
voice1	I understand there is a heavy call flow and our voice support agents are helping our valuable customers like you.
prof1	Please be advised that this is a professional chat. I am sorry to inform you that, I will be forced to disconnect the chat if you are 

using un parliamentary words.
probe1	I would request you to please elaborate the issue, so that I can have the complete understanding of the issue, this would help me 

to assist you better.
probe2	Could you please eloborate the issue, so that I can assist in the best possible way I can.
probe3	Could you please elaborate your issue so that I can assist you better.
probe4	I am sorry, but I did not understand your concern. Please elaborate the issue to assist you better.

again1	I'm sure this issue will be fixed. However, if you still experience the same issue then next time when you connect to the chat 

session, please inform the technician to view your previous chat history. As we document each and every transcript of the chat sessions. 

So even if you contact us again the next technician will be able to read your chat history and notes to continue troubleshoot from there. 

now1	Now send a test email to yourself and check it. 
may0	In order to assist you better, please provide me the exact error message you received on the screen. 
may1	May I know the email program are you using to send and receive email messages? Is it Microsoft Outlook, Windows Live Mail 

or Iphone? Or is it Web Mail? 
may2	May I know if you are using Web Mail to check your Email or do you use Mac Mail, thundebird or Eudora ? 
may3	May I know if you are using Web Mail to check your Email or do you use Outlook Express/MS Outlook/Windows Live 

mail/Windows mail/Thunderbird/ Mac Mail? 
may4	Let us check with the settings first.

know1	Let me know once you are done with the above steps.
know2	After performing the above steps. Check for the issue and let me know the result so that we can perform other troubleshooting 

steps to fix the issue.
know3	Let me know once you are logged in.
know4	Let me know once you are logged in with Temporary Password

hold1	Please allow me 2 minutes while I check the information for you.
hold2	Please allow me 2 minutes while I work on the issue. 
hold3	Please allow me 2 minutes of your precious time while I check that for you.
hold4	Thank you for being on hold, I appreciate your patience.
hold5	I am sorry this is taking more time than usual, please allow me more two minutes.
hold6	It is taking much longer than it is expected. Please bear with me.

help1	I understand your concern, I will help you to resolve the issue, to the best of your satisfaction. 
help2	I will definitely help you with the issue.
help3	Sure, I am glad to assist you with this regard.
help4	I will certainly help you in resolving your issue.
help5	I apologize for the inconvenience caused. I would be glad to assist you regarding this issue.
help6	I'm sorry to hear that you are facing an issue. I'll be happy to resolve that for you.
help7	I'm sorry to hear that you are facing an issue. I'll be happy to help you with that.

free0	You need to delete the unwanted emails from inbox, sent, drafts, personal folders and lastly from trash folder of webmail to free 

up the space on webmail. 
error1	May I know the exact error message displayed when you try to send and receive the email messages?
error2	In order to assist you better, please provide me the exact error message you received on the screen.
glad1	I'm glad I was able to help you with the issue.
it1	It was my pleasure assisting you today. 
it2	It was nice chatting with you.
und3	We don't like to see our customers upset and inconvenienced. We always strive to create a positive customer experience.
und4	The same thing happened to me only recently, let me try and see what I can do to rectify the problem.
sorry1	I am sorry to know that.
hit1	I will go ahead and send data string codes to your modem which will clear the modem's internal memory and Network Traffic 

that may cause internal problem, this will improve your Internet connection.
hit2	I've successfully sent data string codes to your modem to refresh the services. 
hit3	I will go ahead and send data string codes to your cable box which will clear the box's internal memory and Network Traffic 

that may cause internal problem.
hit4	I've successfully sent data string codes to your cable box to refresh the services. 
ver1	For security reasons, may I have the PIN on your account?
ver2	For security reasons, may I have the last four digits of SSN please?
ver3	For security reasons, may I have the account holder's name, Address, Customer code and Phone number please?
tr1	You've reached Texas technical support. Let me transfer the chat to dedicated department for further assistance.
tr2	You've reached New York technical support. Let me transfer the chat to dedicated department for further assistance.?

face1	Your feedback will put a great smile on my face.

reset4	I suggest you please reset the email password at : https://selfcare.rr.com
reboot1	Please reboot your modem by unplugging power cord from the electrical outlet, wait for 30 seconds and plug it back in.
reboot2	Please reboot your cable box by unplugging power cord from the electrical outlet, wait for 30 seconds and plug it back in.
boot1	During the boot process, several things will display on the front panel of the box: BOOT, APP, AIT and OCAP. Finally, the box 

will display a count down from L-8 to L-1.

mail1	Please click the 'envelope' icon top left of the chat. The conversation will be emailed to you. 

pro1	How long have you been experiencing the issue?
pro2	Were there any changes made from the time it was working fine?
pro3	Is the phone line connected directly to modem or the wall jack?
pro4	Is the issue with wired connection or wireless?
sec1	If you are thinking about the chat security, I would like to inform you that you are currently in a secure TWC LiveChat session, 

in direct contact with a TWC LiveChat Technical Support Representative and this is a secure TWC site with 128 bit encryption which 

does not allow any third party to access the information of our Chat conversation.
act1	Please activate the modem at http://elb02.activatemymodem.com/landing
res1	I've checked the information for you.
res2	I see that the signals are good from our end.
sales1	855-892-1002
cncl	888-892-2253
cncl1	800-222-5355 
promo	1-888-892-2253 
cblcard	1866-532-2598 Hours 8 AM to 11 PM
mail1	You can mail or print this chat transcript by clicking on the envelope or printer icon on the top of this window.
cal1 Please provide me your preferred contact number for me to schedule the call back.
cal2 I have successfully scheduled a call back and you will be informed as soon as the maintenance is done.
bis1	I am 200% sure that the issue will be fixed. I am taking the complete ownership. I will ensure everything is taken care up to 

your satisfaction, for me your satisfaction matters the most. I have resolved similar issue for the other customers too so I am pretty 

confident that we can resolve this issue.
bis2	It is my responsibility to put a smile on each and every customer's face by resolving their issue as quickly as possible. It is my 

responsibility to put a smile to each TWC customer with my service. We don't like to see our customers upset and inconvenienced. We 

always strive to create a positive customer experience. The same thing happened to me only recently, let me try and see what I can do to 

rectify the problem.
bis3	Let me see what best can be done to get this sorted out on priority basis.
bis4	We don't like to see our customers upset and inconvenienced. We always strive to create a positive customer experience.
bis5	Trust me, I will do everything I can to get this rightly addressed.

isol	I have successfully isolated he issue from my end now. I see that the firmware and boot version on your modem is outdated and 

the channel wasn't set up properly on modem hence you're experiencing the issue.
chang	I am making the required changes now and increasing the bandwidth on the modem. I've made the required changes in the 

modem's settings and have updated the firmware and boot version on it.

stand1	From my end the issue stands resolve as I have isolated the issue and have provided with the permanent solution for that.
stand2	Just to confirm, since the issue needs to be addressed by a technician it is resolved from our end by successfully scheduling a 

technician
note1	I have also left a note regarding this on your account so that you don't need to repeat all over again when the technician arrives 

at your place.
note2	I have left a detailed note on your account so that you won't have to explain everything again in detail. 
resp1	It is my responsibility to put a smile on each and every customer's face by resolving their issue as quickly as possible.
resp2	I wish I have customer's like you in future for support. You were really good through out the chat today.
fore1	My foremost priority over this chat is to see the customer happy with my assistance
smile1	It is my responsibility to put a smile to each TWC customer with my service
resp3	It is my responsibility. I will try my best to fix this issue by the end of this chat.
supp1	Thank you for your support and we are pleased to have customers like you.
limit1	I would have loved to sort this out for you right away. I trust you'll understand my limitation, that I am a member of Online 

Customer Services Team and have limited access to certain tools and applications due to which I am unable to process your request from 

my end.
cls1	If you have a second, please give your feedback on how I did by clicking 'END CHAT'.We appreciate you contacting us today! 
msur2	Please click on "NEXT" button to get the feedback form once we are done. It is important for me.
msur3	Please click on the 'NEXT' button for your valuable feedback. 
msur4	I hope I've answered all your concerns. If you would like to take a brief survey, please wait for the chat session to end and the 

survey will load in this same window. We greatly appreciate your thoughts.
cls3	Thank you for chatting us. We value you as a customer and are here to assist you 24 hours a day, 7 days a week. 
ask1	It will only take a few minutes to straighten this out. Is it ok if I ask you a few questions so I may begin to get this resolved for 

you?
mac1	May I have the MAC address (C mac) of your device? It is located at the rear of the device on a sticker.
own1	As a TWC representative I will take the ownership and personally follow up this for you.
own2	You have reached the right person and I will be glad to take care of it for you.
how1	How are you doing, ?
gr1	Great to hear that from you. 
fab1	I am doing fabulous, thank you for asking.

faq1	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic from the "Select a Topic" dropdown box. 
faq2	Pick a sub topic relevant to your question, then select the question that you want an answer to, from the list of questions listed 

on the right hand side.
faq3	If you aren't able to find the answer in the list of FAQs, please contact us for further assistance.
must1	Good to know information, you can login online at https://www.timewarnercable.com and click on "My Account" to 

troubleshoot your TWC services, check for outages, pay your bills and manage your DVR. 
must2	You can also upgrade your services, manage your phone and its features, set digital phone features, check your order status, 

check for outages, view your billing, make a payment, reset your TWC equipment and check tech visit updates by visiting our website 

https://myservices.timewarnercable.com/	
fut1	The above link is for your future reference.
self1	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "Internet" from the "Select a Topic" dropdown box. Pick a sub topic as Internet Plans, then select the question that you 

want an answer to, from the list of questions listed on the right hand side. 
self2	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "TV" from the "Select a Topic" dropdown box. Pick a sub topic relevant to your question, then select the question that 

you want an answer to, from the list of questions listed on the right hand side. 
self3	You can also view our FAQs and troubleshooting tips by visiting http://www.timewarnercable.com/en/support/all-faqs.html and 

picking a topic as "Phone" from the "Select a Topic" dropdown box. Pick a sub topic relevant to your question, then select the question 

that you want an answer to, from the list of questions listed on the right hand side. 
online1	I will be more than happy to stay online for you. Please let me know once you're done.
done1	I've completed all the troubleshooting steps I'm able to assist you with. I would like to schedule a technician visit so the 

connections and devices can be physically checked and the issue isolated.
speed1	I see that there is no outage affected in your area. The signals are good from our end. The good news is that I can fix this issue 

now. I just have to send the signals to boost signal levels and change the channel frequencies on routers page. I am going to set up the best 

channel frequency on your modem. This channel frequency will give you high speeds with consistent connectivity. Can I set up channel 

frequency for you? 
speed2	Please allow me a minute to set the best channel frequency for you. I have successfully set the best channel frequency on your 

modem now and have also sent the signals. To update this on your end, let me reboot the modem on the network so we will be all set. This 

will disconnect the connection for a moment and you will be online again with a minute or two.
speed3	Rest assured, everything has been taken care of. I will ensure you won't have to get back to us for the same issue again. The 

issue will be fixed permanently. I will reboot the modem after we end the chat session.
speed4	I request you to check the speeds in next 10 minutes and everything will be in place there after. I'm glad that I was able to 

resolve your issue. Is there any thing else I can help you with?
bas1	I will help you with some basic troubleshooting steps to resolve this issue. 
o1	I see that there is no outage affected in your area. The signals are good from our end.
reset1	Let us try to factory reset the modem, I request you to press the reset button which is located at the rare end of the modem. 

Please press it for 10 seconds and then release. All the settings will be restored to it's default.
upd1	Let me quickly go through your account and check for the updates.
2min	Please allow me 2 minutes of your precious time.
care1	I will ensure everything is taken care before we end this chat session.

status1	Please keep me updated with the status. 
status2	May I know the status please?
status3	Thank you for updating me the status.
goal1	My goal was to provide you with an excellent customer service. Did I do that for you today? 
assure1	I assure you that you will not experience any issues going forward. I will ensure you won't have to get back to us for the same 

issue again. The issue will be fixed permanently. 

y1	You're welcome.
take1	Take care of yourselves!!!
name1	May I have your name please? 
keep1	Please keep me update with the status of reboot.
smile1	I assure you that, at the end of the chat I will make you smile. 
over1	We have done it. Thank you for your support. 
fix1	Should I close our conversation as issue resolved, then?
g1	Great, I am very happy to know that.
ef1	I really appreciate your efforts in doing that.
ef2	I really appreciate all of your efforts in troubleshooting the issue.

und1	I do understand that and trust me if I were you I would have feel the same.
und2	I know what you're trying to say, I will do my best to help you out.
und3	The same thing happened to me only recently, let me try and see what I can do to rectify the problem.
und4	I completely understand the situation. I agree with you in this regard.
und5	If I were in your place, I would feel the same if I come across such situation.

nice1	It was pleasure assisting you today. Thank you for being so polite and patient through out the chat.
nice2	I want to make sure you have the best experience possible today with our chat service. 

longer1	It will not be longer because we know our customers wouldn't be happy with it.
pre1	May I have your preferred contact number to update you?
lot1	We have lot of customer's who are impacted, but it is for the improvement of services only.Hope you understand. 

Outages
out1	Sure, let me check the line for you. 
out2	I have checked the updates for you.
out3	We've got an update that there is an outage affected in your area.
out4	As we're already aware of this issue, our engineering team is working on it to get this issue fixed.
out5	We are confident that our engineers will have this issue resolved.
out6	We don't have estimate time of resolution. 
out7	If you would like, I can go ahead and set up the call back from our automated system, once the issue is fixed. 
out8	Could you please help me with the preferred number to arrange a call back?
out9	I've successfully set up the outage call back for you, you'll be notified once it is fixed.
out10	There is no issue with your equipment, there are lot of customers who are impacted due to this, once the outage is fixed all the 

services will be up and running in your area automatically.
out11	I am glad that I was able to answer all of your concerns by helping you with the updates.
out12	I hope you are happy with my quick assistance.
out14	 I do sincerely apologize for this situation and I have a thought which might help you get connected to finish your work.  With 

your Spectrum Internet service you do have free access to Spectrum WiFi Hotspots in your area.  You can quickly locate the nearest 

Hotspot location by using the Spectrum WiFi Finder App with your smartphone or tablet.
PARAPRASHING
*************************************************************************************
pi	I understand you are facing an issue with Internet services.
pis	I understand that you're facing an issue with slow browsing speeds.
pii	I understand that you're facing an issue with intermittent connectivity.
pw	I understand that you're unable to connect to WIFI.
pt	I understand you are facing an issue with Cable services.
pm	I understand you are facing an issue with Missing channels.
ppic	I understand that you're unable to see a picture on your TV.
po	I understand you are facing an issue with On demand.
pd	I understand you are facing an issue with DVR recordings/services.
pp	I understand you are facing an issue with Phone services.
pno	I understand that you're unable to hear a dial tone.
pact	I understand that you would like help activating your modem.
iund1	I understand you are facing an issue with . Correct
iund2	I understand you would like to know 
apo	I apologize for the inconvenience, I will be glad to help you with the issue.
reac	You've reached the right person. I'll take care of this for you right away.
apho	In order to bring up your account details, could you please provide me the phone number that you have used for registering 

with the TWC account? 
aco	In order to bring up your account details, could you please provide the account number registering with the TWC account? 
addr	In order to bring up your account details, could you please provide me your name and full service address that you have used 

for registering with the TWC account? 
time1	Please allow me 2 minutes to get some information on your account, this will help me to assist you better.
revi1	Please give me 1-2 minutes while I review the previous chat details.

revi2	Please allow me a moment while I go through your previous chat conversation so theres no need for you to explain the issue 

again.

min	Please respond for every two minutes. It will help us that we are still connected.
2min	Please give me 1-2 minutes while I work on the issue.
3min	I'm still in the process of checking the information. It will take two more minutes. I apologize for the delay and appreciate your 

continued patience.
4min	I see this process is taking some extended time, hence I will appreciate you waiting, as your cooperation will help me a lot.

info1	I have checked the required information.
sup1	Please give me two minutes while I discuss your issue with my supervisor.
own1	I'll take complete ownership to resolve the issue for you.
own2	I am sorry about the trouble that you are going through with your Internet. I will take complete ownership and get your issue 

resolved for you.

ver1	Thank you for verifying the account. Your account has been authenticated. This is to make sure that I have the right account 

and avoid identity theft.
ver2	We value your account security and it is necessary to authenticate whenever sensitive information is involved. I hope you 

understand.

notice1	I really appreciate you for getting this to my notice. 
notice2	Thanks for alerting us to the bad service you have received. I assure you, I'll do everything I can to help you with this.

efforts1	I really appreciate your efforts.
efforts2	I really appreciate your efforts and time towards resolving the issue.
taketime	Please take your time. I'll be happy to stay online with you .

pleasego	Please go ahead. I'll be happy to stay online with you.

prech	I have read your concerns and I will try my best to help you with the issue

igno	Please ignore the above message as it was triggered automatically.

from1	I assure you that, you won't face any issues again.

credit1	Sure, I'll go ahead and process the credits on your account right away for the non-service period.
credit2	I have successfully processed the credits on your account for the non-service period.

spanish1	You have reached English support team. Are you comfortable with English or I can connect you to Spanish support team.

repeat1	I'm sorry to hear that you've had to contact us repeatedly about the same issue. Let me see what best I can do to resolve this for 

you to avoid the possibility of this reoccurring.
repeat2	Please accept our sincere apologies for the inconvenience you have experienced. I appreciate the time you have taken to bring 

this to our attention and for giving us an opportunity to rectify the situation. 

PROBING
**************************************************************************************************************

*******************************************

prob	Let me ask you a few questions to determine the possible cause of the issue. That will help me to isolate the issue.
how1	How long have you been facing this issue?
change1	Were any changes made since the time it was working fine?
worwc	Is the issue on wired connection or wireless connection?
cable1	Please check if all the cables are finger tight and secure.
cable2	Also ensure all the cables are connected secure and hand-tight.
3party	Just to confirm, is there any external router connected to the modem?
light1	Please provide me the light status of the modem.
light2	Is the online light stable/solid on the modem?
odorad	Are you facing this issue with one device or all the devices connected to it?

error1	Do you see any error message on the screen?
error2	May I have the error code or error message that displayed on the device when you try to connect to your Internet? 
ocorac	Are you facing this issue with particular channel or all the channels?

enq1	Could you please elaborate your query? That will help me to assist you better.
enq2	Could you please be more specific about the issue? That will help me to assist you better.

input1	Please check if you have selected proper input on your TV.
source1	Please press the 'Source' button on your TV remote, keep pressing the 'Source' button until you get the picture.

mnm1	Could you please provide me the make and model of the modem?
mnm2	Could you please provide me the make and model of the cable box?

bill1	I know that bills can be confusing sometimes. Let me have a look at your account to help you better understand what you're 

being billed for.
bill2	I know that bills can be confusing at times. I am a customer too and I would like to make sure I understand everything that I am 

paying for. Let me go ahead and pull up your account so we can discuss it.

off1	I see that the modem is showing as offline and unreachable on my side. Please reboot the modem by unplugging the power cord 

for 30 seconds and then plug it back in. Also ensure all the cables are connected secure and hand-tight.

speed1	Could you please run a speedtest on hard wired connection by visiting this link: 

http://www.timewarnercable.com/en/support/speed-test.html
speed2	Please let me know the results once it is done. I'll be happy to stay online with you.

ping1	Let me quickly run a ping test on the modem and check if there is any packet loss. Please give me 1-2 minutes.
ping2	Our diagnostics show that the modem is online and is receiving good signals. 

checkom	Could you please check if all the cables connected from the wall outlet to the modem and from the modem to the devices 

(laptop/computer) are securely connected and are hand tight?

hard1	I recommend you to factory reset the modem. The factory reset button is usually found at the back of the Router/Modem. 

Using a paper clip or pin, push the reset button in for ten seconds and release.
hard2	Once you hard reset the modem, all the settings and WiFi credentials will set back to default.
history1	I would also suggest you to delete the cookies, cache and browser history from the Internet settings to get faster Internet 

services.

persist1	I will walk you through some troubleshooting steps and try to fix the issue. However, if the problem still persists we may need 

to replace your equipment or if not, dispatch a technician to resolve it.

persist2	I am sure the issue will be resolved. If the issue still persist, I will go ahead and schedule a technician visit for you.

notto	Not to rush you but are we still connected?

issuefix	From my side the issue is fixed as I've sent the signals which will fix the issue and made necessary changes from my side.

twcdigi	Now the TWC as digitalized. Without the cable box you cannot watch the channels. I suggest you to please buy a new cable 

box or I can order from my end for you.

tel1	I request you to please unplug the cable form TEL1 port and plug it back again to the TEL1 port after 30 seconds.

note1	I have also document a detailed note about this issue on your account.
note2	I have left a detailed note on your account so that you don't have to explain everything again in detail, and the next agent will 

be able to assist you further.

env1	To avoid a situation like this in the future, I would suggest that you make a note of your user name and password.

env2	Please click on the 'envelope' icon top left of the chat window. The chat conversation will be emailed to you.

ci1	Resetting your modem often times allows your devices to properly reconnect to TWC.
ci2	Next time this happens, try resetting your modem by unplugging the power cord like we just did.

cc1	By rebooting your cable box we can confirm the signal is being sent to your TV.
cc2	If this happens again, just turn your box off and on again like we just did.
cc3	If this happens again, try rebooting your cable box by unplugging the power cord like we just did.

rio	I'll go ahead and reset the modem from my end for you. I am sure that this will be very helpful for you.

expi1	Internet connectivity issues, will be caused due to the networking traffic junk. As I have sent the refresh signals and we 

rebooted the modem the issue got resolved.
expc1	Cable connectivity issues, will be caused due to the networking traffic junk. As I have sent the refresh signals and we rebooted 

the box the issue got resolved.

expp1	Phone connectivity issues, will be caused due to the networking traffic junk. As I have sent the refresh signals and we rebooted 

the modem the issue got resolved.

GTA
**************************************************************************************************************

************************************************************************************
gta1	Let me take control of your computer and check the settings for you. I will try my best to resolve your issue as soon as possible.
gta2	I would like to begin a GoToAssist screen sharing session which enables us to share control of your computer's desktop. I 

believe it will help resolve your issue. You have the option to resume full control or to stop or pause screen sharing at any time. Is it alright 

with you for me to begin?
gta3	Please click on the above link >> Run the application to start the sharing session.
gta4	I will be away from this chat session while I log into your system to resolve the issue and will return to the session as soon as I 

have completed it or when I need some information from you. 
gta5	I suggest you please not to touch the mouse and keyboard while I work on the issue and watch me working on your computer to 

help you follow the same steps when you face the same issue in future.
gta6	Please don't close the remote sharing session, I will do it at our end. 
gta7	Please close all the browser windows (except chat window) and click on above link to start remote session. Once the remote 

download is finished you will get the prompt to run the remote session. Please click on Run to start the session.
gta8	Since, the issue is resolved, I will go ahead and disconnect the Screen-Sharing from my end. Please stay with me for a moment 

while I confirm the sharing session has been disconnected. Please don't close the chat window.
gta9	I have disconnected the Screen-Sharing Session and I am no longer able to view your screen.

AUTHINTICATION
acco0	"If your attempt to login has failed multiple times, you will automatically be locked out of your account for 15 minutes. I would 

recommend waiting for it to automatically unlock and trying to login, before you contact us again, as this usually resolves the issue."

acco1	I'm more than happy to assist you. To get us started, I'm going to need to verify some information. 

acco2	In order to reset your My account password, I need to authenticate you with few details. Please provide me with the below 

details. 1.Account holder name 2.Serviceaddress. 3.Phone number. 4.Customer code. A Customer Code is a four digit number located at 

the top right of your bill, under the account number.

acco3	Could you please help me with the preferred email so that I can send you the reset password link to your mail?

mac1	Could you please provide the MAC address of the equipment? You can find the MAC at the bottom of the modem.

mac2	Please provide me with the mac address which is a 12 digit hexadecimal number labelled as CM MAC. It can be found on the 

back of the modem.

mac3	Please provide me with the MAC address of the modem which is printed on a sticker, at the bottom or the back of the modem 

as 12 hexadecimal characters (0-9, A-F). It will be in the form of CM MAC or RF MAC or HFC MAC ID. 

ssn1	In order to help me authenticate the account, could you please provide me with the last four digits of the SSN? 

pin1	Please provide me with the PIN number. The PIN is a 4-digit number that was selected by you and it allows me to authenticate 

the account.

acc1	For security reasons, could you please confirm the account number?

SELF SERVE
*************************************************************************************
abv	The link above is for future reference.
mean1	In the meantime good to know information, I will provide you some links for future help. 
mean2	Meantime good to know information, you can always visit our website. 

myacc	Meantime good to know information, you can always visit our website https://myservices.timewarnercable.com/ and sign into 

your account for setting digital phone features, checking order status, checking for outages, billing, making payments, resetting Spectrum 

equipment and check technician visit updates.

myacc2	Just so you know, you can easily access multiple features like My Account and Voice Zone using the My Spectrum app 

available on Android and IOS based devices. To download the free My Spectrum app, copy this URL for quick access to the app after our 

chat has been completed: https://mytwc.mpsvcs.com/

smyacc	If you have any issues logging in to your account in future, you can directly visit this link to reset your password: 

https://registration.timewarnercable.com/password/reset

smyacc2	You can also access this link to reset the password on your own: https://registration.timewarnercable.com/password/reset

sbill	For future reference, you can view your bill using the My TWC App from your mobile phone or tablet and can also check the 

complete breakdown of your current bill and the last 5 bills from the My Account feature on our website. To access your account, please 

login to: https://myservices.timewarnercable.com

TV
**************************************************************************************************************

*************************************

stv	For future reference, please access this link to troubleshoot issues with your Cable TV: 

https://www.timewarnercable.com/en/support/tv/step-by-step/tv-picture-quality.html

smissing	For future reference, please access this link to troubleshoot issues with your TV channels: 

https://www.timewarnercable.com/en/support/tv/step-by-step/tv-picture-quality.html

sdemand	You can also find instructions to order or troubleshoot issues with your On Demand program at 

http://www.timewarnercable.com/en/support/tv/topics/on-demand.html

sremote	For step by step instructions on how to program your remote, you can always visit : 

http://www.timewarnercable.com/en/support/faqs/faqs-tv/tvtroubles/how-do-i-program-my-cable-remo.html

sdvr	You can also troubleshoot DVR issues on your own by following the steps listed at 

http://www.timewarnercable.com/en/support/faqs/faqs-tv/dvr_digita/what-should-i-do-if-my-dvr-is-.html

sline	You can also find information relating to channel line-ups and TV listings at http://www.timewarnercable.com/en/tv/channels-

lineup.html

spic	For future reference, please access this link to troubleshoot issues with the picture quality of your TV channels: 

https://www.timewarnercable.com/en/support/tv/step-by-step/tv-picture-quality.html

ssound	For future reference, please access this link to troubleshoot issues with the sound quality of your TV channels: 

https://www.timewarnercable.com/en/support/tv/step-by-step/tv-picture-quality.html

INTERNET
**************************************************************************************************************

**********

sint	You can also troubleshoot this issue with no internet connectivity on your own by following the steps listed at: 

http://www.timewarnercable.com/en/support/internet/step-by-step/internet-troubleshooting.html

sinter	In the future if you ever have an issue with intermittent internet connectivity, you can troubleshoot it on your own by visiting: 

http://www.timewarnercable.com/en/support/internet/step-by-step/internet-troubleshooting.html

sspeed	In the future if you ever have an issue with slow internet speed, you can troubleshoot it on your own by visiting: 

http://www.timewarnercable.com/en/support/internet/step-by-step/internet-troubleshooting.html

swifi	To Setup and Manage Home WiFi, please visit: http://www.timewarnercable.com/en/support/internet/topics/wireless-

networking.html

swifib	If you would like to maximize your home WiFi performance, I would suggest that you take a look at our top 10 tips: 

http://www.timewarnercable.com/en/support/internet/topics/home-wifi.html

swifik	For instructions on how to view your Wi-Fi name or password, you can always visit 

http://www.timewarnercable.com/en/support/faqs/faqs-internet/homewifi/how-do-i-change-my-wifi-settings1.html

swific	For instructions on how to change your Wi-Fi name or password, you can always visit 

http://www.timewarnercable.com/en/support/faqs/faqs-internet/homewifi/how-do-i-change-my-wifi-settings1.html

smodem	You can request for a modem swap or upgrade at http://www.timewarnercable.com/en/support/internet/topics/modem-

swap.html	

swifihot	To find the nearest TWC Wifi hotspot you can visit: http://timewarnercable.cellmaps.com/WiFi.html

PHONE
*********************************************************************************************

sp	Should you face any issues with your phone connection in the future and wish to troubleshoot it on your own, please visit: 

http://www.timewarnercable.com/en/support/phone/step-by-step/phone-troubleshooting.html

spno	In case you are unable to hear a dial tone on your phone in the future, we also offer step-by-step phone troubleshooting 

instructions at: http://www.timewarnercable.com/en/support/phone/step-by-step/phone-troubleshooting.html

sppoor	Should you face any issues with the quality of your phone calls in the future and wish to troubleshoot it on your own, please 

visit: http://www.timewarnercable.com/en/support/phone/step-by-step/phone-troubleshooting.html

sp2	To find out the great features available with your phone service, please visit: 

http://www.timewarnercable.com/en/support/phone/topics/calling-features.html

EMAIL
*********************************************************************************************

semail	For any assistance with your TWC Email, you can always visit http://www.timewarnercable.com/en/support/internet/step-by-

step/twc-mail.html

semailu	To find out how to retrieve your TWC Email Username on your own, please visit: 

http://www.timewarnercable.com/en/support/internet/step-by-step/twc-mail.html#

semailp	To find out how to retrieve your TWC Email Password on your own, please visit: 

http://www.timewarnercable.com/en/support/internet/step-by-step/twc-mail.html#

semailp2	For security reasons, we'd recommend that you change your password. To do this, please log on to webmail.twc.com

smac	To download and install McAfee antivirus or family protection, please visit 

http://www.timewarnercable.com/en/support/internet/step-by-step/mcafee-antivirus-and-family-protection.html

soutlook	To set up your email account on Outlook or any other client email, please visit 

http://www.timewarnercable.com/en/support/faqs/faqs-internet/e-mailacco/incoming-outgoing-server-addresses.html	

sstore	To locate your nearest local TWC store, please visit http://www.timewarnercable.com/en/support/twc-stores.html	

smove	If you are moving to a new location and would like to see if our My Move service is available in your area, you can visit 

http://www.timewarnercable.com/en/support/twc-my-move.html or call our team at 1-855-235-4220.	

stech	You can check the appointment details and wish to reschedule or confirm your appointment date and time, please login to your 

online account at: http://www.timewarnercable.com/en/residential-home/support/faqs/faqs-account-and-billing/appointments/how-can-i-

check-the-status-of-.html Here, you can cancel, reschedule or check the status of your work order/appointment.

sapp	To find out what features our available with our TWC TV app, please visit 

http://www.timewarnercable.com/en/support/faqs/faqs-account-and-billing/mytwcapp/what-can-i-do-with-the-my-twc-.html

smodem	Please visit this link to view a list of modems approved for TWC use: 

http://www.timewarnercable.com/en/support/internet/topics/lease-or-buy-modem.html

VOICE MAIL	
**************************************************************************************************************

***************************************************
svsetup	For step by step instructions on how to set up the voice mail feature, please visit: 

http://www.timewarnercable.com/en/support/faqs/faqs-phone/voicemail/how-do-i-set-up-my-voicemail.html

svoicer	For step by step instructions on how to retrieve your voice mail messages, please visit: 

http://www.timewarnercable.com/en/support/faqs/faqs-phone/voicemail/how-do-i-retrieve-my-voicemail.html

svoicea	To find out what your Voicemail Access Number is, please visit: http://www.timewarnercable.com/en/support/faqs/faqs-

phone/voicemail/what-is-my-voicemail-access-nu.html

svoicepin	To set your Voicemail PIN, please access the below link to know the PIN for future reference : 

http://www.timewarnercable.com/en/support/phone/topics/voicemail-setup.html

TWC APP
**************************************************************************************************************

***************************************************
sabil	You can also view your bill/manage payments and easily contact TWC for support using the My TWC App from your mobile 

phone or tablet. To download the free My TWC app, copy this URL for quick access to the app after our chat has been completed: 

https://mytwc.mpsvcs.com/

saupgrade	You can also upgrade your services from the My TWC app. Please copy the URL (https://mytwc.mpsvcs.com/) to quickly 

access and download the My TWC app.

savoice	In the future, if you want to access and manage your phone settings and voicemail online, you can access VoiceZone using the 

My TWC App from your mobile phone or tablet. To download the free My TWC app, copy this URL for quick access to the app after our 

chat has been completed: https://mytwc.mpsvcs.com/

savoicer	You can now view incoming call logs, listen and manage voicemail messages and manage Call Forward settings on the go using 

the My TWC App. Just copy this URL to access and download the app on your device: https://mytwc.mpsvcs.com/

savoicea	You can now easily find out your TWC home phone number using the My TWC App. Just copy this URL to access and 

download the app on your device: https://mytwc.mpsvcs.com/

sadvr	You can now easily schedule/manage the DVR recordings using the My TWC App. Just copy this URL to access and download 

the app on your device: https://mytwc.mpsvcs.com/

saremote	You can now easily navigate TV channels using the My TWC App. Just copy this URL to access and download the app on your 

device. https://mytwc.mpsvcs.com/

saall	You can also troubleshoot Cable TV/Phone/Internet issues using the My TWC App from your mobile phone or tablet. To 

download the free My TWC app, copy this URL for quick access to the app after our chat has been completed: 

https://mytwc.mpsvcs.com/

saall2	You can also troubleshoot all of your equipment/easily manage your TWC appointments and contact TWC for support using 

the My TWC App from your mobile phone or tablet. To download the free My TWC app, copy this URL for quick access to the app after 

our chat has been completed: https://mytwc.mpsvcs.com/

sastore	You can easily find a TWC Store and view where it is on the map using the My TWC App from your mobile phone or tablet. To 

download the free My TWC app, copy this URL for quick access to the app after our chat has been completed: 

https://mytwc.mpsvcs.com/

sahot	You can easily use TWC WiFi when you are on the go and contact TWC for support using the My TWC App from your mobile 

phone or tablet. To download the free My TWC app, copy this URL for quick access to the app after our chat has been completed: 

https://mytwc.mpsvcs.com/

sachannel	You can easily view your Channel Guide and contact TWC for support using the My TWC App from your mobile phone or 

tablet. To download the free My TWC app, copy this URL for quick access to the app after our chat has been completed: 

https://mytwc.mpsvcs.com/

satech	You can view, cancel or modify your appointment from the My Account tab in My Account: 

https://myservices.timewarnercable.com/ or by downloading the My TWC app using https://mytwc.mpsvcs.com/ 

saoutage	You can also check for known outages or check the status of an outage in your area by logging in to the My TWC App 

(https://mytwc.mpsvcs.com/) or My Account (https://myservices.timewarnercable.com). You may also contact our Local Office number to 

receive updates from the IVR.

linkma	https://myservices.timewarnercable.com/
linkst	http://www.timewarnercable.com/en/support/twc-stores.html
linkweb	https://webmail.twc.com/
linkreg	https://registration.rr.com/
linkreg2	http://elb02.activatemymodem.com/landing

FRUSTRATING
*************************************************************************************
fust1	I understand you're upset with the issue. I'll do my level best to resolve the issue for you.
fust2	I understand you're really concerned. I'll do my level best to resolve the issue for you.
fust3	I am sorry to hear that, I'll be glad to help you with the issue.
fust4	I have read your concerns, I'll do my level best to resolve the issue for you.
fust5	I understand this might have been frustrating for you, I will try my best to get this all fixed today.
fust8	I can completely understand. If that happened to me I'd be really upset too. I can imagine how frustrating that must be.
bis1	We don't want to see our customers upset and inconvenienced. We always strive to create a positive customer experience. 

bis2	My foremost priority over this chat is to see the customer happy with my assistance.

upset1	I understand this entire situation has upset you.
upset2	It's reasonable to be upset when you have to spend your personal time on this issue.
upset3	I'm sorry you have had such a bad experience. I assure you, I'll do everything I can to help you with this.

coop	Your patience and co-operation is greatly appreciated.

TECHNICIAN
****************************************************************************************

tech1	In this case I will not waste much of your time doing the same steps again.
tech2	I have a better resolution to fix this issue permanently.
tech3	As all the trouble shooting steps are performed. I see that the box has to be tested manually so that I will send an expert 

technician to your home to fix this issue. 
tech4	I assure you that the technician will definitely fix this issue with his advanced tools also if required he may also replace a new 

box for you at your place, Is that fine with you? 
tech5	Let me check for the earliest slot available to send a technician to your place. 
tech6	I see that the earliest time for the availability of our technician team to reach your home is on at . Is this time ok for you? 

tech7	You have an option to receive the automated reminders by Phone, E-mail or Text. Please let me know your preferred method of 

contact so I can set up the automated reminders accordingly.
tech8	What would be the preferred number for the technician to call you before he visit your place? 
tech9	Our normal service calls are free of charge. In the case of service issues that can typically be resolved over the phone or in 

situations where service issues should are unrelated to TWC, we charge a service fee of $39.99. When the technician arrives at your home 

on (date), he/she will be able to determine the cause of the problem and let you know if there will be a service fee associated with the 

home visit." 
tech10	The first reminder occurs the night before your appointment. The second reminder occurs on the day of your appointment, 

approximately one hour before your scheduled time. The final reminder occurs once your technician is on the way to your residence. Also, 

please make sure there is someone 18 years and older at the location.
tech11	Please keep in mind that since you have chosen to be contacted by phone, the TWC number you will see displayed is (800) 

892-2253. You may also see "TIME WARNER CAB" depending on the device you are using. Please be sure to answer these calls, or 

review any messages left by this number.
tech12	I have successfully scheduled a technician visit to your place and I have also made a detailed note regarding the issue on your 

account that will help him to assist you better. 
tech13	I have also document a detailed note about this issue on the account for the technician to refer. Rest assured. If our technician 

finds an issue with the modem, he will replace it for you. If there are any signal issues, he will fix that for you as well.
tech14	I assure you that once the technician visits your home he will definitely fix the issue. 
tech15	You can check the appointment details and wish to reschedule or confirm your appointment date and time, please login to your 

online account at: http://www.timewarnercable.com/en/residential-home/support/faqs/faqs-account-and-billing/appointments/how-can-i-

check-the-status-of-.html Here, you can cancel, reschedule or check the status of your work order/appointment.
tech16	I am glad that I was able to provide you the resolution by scheduling the tech visit.
tech17	I assure you that you will enjoy all your services, once the technician visits your place. 
tech18	The issue stands resolved as per the chat's prospective as I have provided all the required information that I have.
techcharge	You will be charged $39.99 only if you caused any damage to cable box or to the wires and is not from TWC end.

wall1	If the wall outlet in that room is not activated or set up previously by a technician: then, you will need a technician to visit and 

install the outlet for you and then connect a new box to it. (this will incur installation charges) 
wall2	However, if you already had TWC services at that wall outlet earlier, then you can have a new box shipped to you or visit the 

store and directly plug it in.

CLOSING
*************************************************************************************
smile1	It is my responsibility to put a smile to each TWC customer with my service.

smile2	It is my responsibility to put a smile on each and every customer's face by resolving their issue as quickly as possible. 

primary1	Your satisfaction is my primary concern.

primary2	We value you as a customer and your satisfaction is most important to us. 

summ	I am glad that I was able to resolve your issue on this chat today. Is there anything else I may assist you with today?

reso1	I just want to make sure that you're happy with the resolution part given by me on this chat. Are you happy with the resolution 

part given by me on this chat?

conc	Please let me know if you have any more query's or concerns for me.

anyconc	Are there any other questions or concerns I may be able to address for you today?

ir1	Was I able to answer all your questions to your satisfaction from my end? 
ir2	I am glad that, I was able to resolve your concern by providing you the required information.

polite1	Thank you for being patient and polite throughout the chat. 

plea	It was my pleasure assisting you today. 

wond	It was wonderful chatting with you today.

goal1	My goal was to provide you with an excellent customer service. Did I do that for you today?

close1	I appreciate your patient approach and cooperation in getting this issue resolved, you have actually been really helpful during 

our interaction. I want to thank you for the same and generous assistance. It has all been possible due to your efforts and co-operation. 
close2	I hope, things are clear now and you understood concisely the information which I provided you. 
close3	It is my responsibility to help you and answer all your questions. You have been so patient and cooperative, appreciate this. It 

has been my pleasure to assist you. Have a wonderful and pleasant day ahead. Bye and take care. 
close4	You have been a wonderful customer to assist. It was pleasure chatting with you. 
close5	I want to thank you for being supportive and for co-operating with me while I had a chance to work on this issue. 
close6	I wish I always have supportive customers like you. 
close7	I am glad to know that. Like I said, I will do my best to fix all the issues my customers have. 

custapp1	Thank you for appreciating, it does make a lot of difference.
custapp2	It's such compliments which help us to serve you better.

cls2	We value you as a customer and are here to assist you 24 hours a day, 7 days a week. 
cls1	Again, my name is Rakesh. Thank you for contacting us. 
cls3	I appreciate the time you have taken to chat with us. Thank you for choosing chat support.

sum1	You contacted us regarding and I am glad that I was able to resolve the issue. Is there anything else that I can help you today??

sum2	Was I able to answer all your questions to your satisfaction and resolve the issue from my end?

dis1	I have not heard from you for a few minutes. Please confirm if we are connected.
dis2	I am afraid this chat might get disconnected automatically due to lack of response.
dis3	I understand that something is keeping you busy at this moment, hence I would go ahead and disconnect the chat session. We 

are closing this chat session due to an extended period of no response. If you still need assistance, please do contact us in the future. We 

value you as a customer and are here to assist you 24 hours a day, 7 days a week.

SURVEY STATEMENTS
**************************************************************************************************************

**************************************************************************
sur1	When this chat ends, I hope you can lend me a moment of your time to answer the post chat survey after you end the chat 

session.   
sur2	Please click on "END CONVERSATION" and the survey will load in this same window.
sur3	Please click on "NEXT" to take a brief survey once the chat ends. Your feedback is greatly appreciated.
sur4	Please click on (X) and click on ''Stay on the same page''. 
sur5	Please take the survey once the chat ends. Your feedback is very important to me. Thank you for your time. 

is5	Would you like me to help you with any other details?
OUTLETS
*************************************************************************************
outlet1	Please give me 1-2 minutes while I work on the issue.
outlet2	I see that the equipment was not arranged correctly on your account.
outlet3	I have made some necessary changes on your account.
outlet4	I rectified that issue and this will fix the issue for you.

4piece		For security reasons, could you please confirm:
1) Name on the account
2) Address on the account
3) Customer code
4) Home phone number

pay1	I see that the services have been temporarily disconnected because of past due  on your account, as you ha ve made the 

payment, I will go ahead and restore the services for you right away.

nopay1	I see that the services  have  been temporarily disconnected because of past  due of $... on your account, I request you to make 

the payment so that I can restore the services for you right away.

out15			Unfortunately, outages are due to unforeseen circumstances such as poor weather conditions, natural 

calamities, construction work, road work, accidents or other incidents that may damage, uproot or sever vital cables and connections in 

some way.

CONVENING

co1	Request you to please support us at this tough time and we won't let our customer's down.
co2	We are working on it ensure our customers are not disappointed.
co3	I know it's not easy to compromise for service when you paying for it, however, this is not something that we did intentionally it 

just happened.
co4	We love all our customers also appreciate their business with us and we doing our best to retain this channel with us.
co5	I know I couldn't do much except giving you the information right now, however, all we need is little time and patience from 

your end and we are hoping quickly to come back with a good news.
co6	Hope you understand my situation I could only give you the information that I could see it on my end.
co7	We did have such issues before and it was your support made us stronger to get over it and this time again we would request 

you to support us and we will do our best please believe in us.
co8	It's customer who makes us who we are and we do value your importance and business with us.
co9	All we need is little time to work on it and request your time and support.
co10	I know you won't be happy with what has happened and we are not too, however, will try and resolve this at the earliest.
co11	We don't wish to see any of our customers upset.
co12	I just started my day with you and hope to let you go with a smile.
co13	Your happiness means a lot to me, Let me assure you that I'll do everything I can to take care of this fo you.
co14	I'll stay online with you until you get connected to them over the phone just in case you may need my assistance.
co15	I've just started my day with you as your my first customer.
co16	I'm sorry to hear that you've had to contact us repeatedly about the same issue. I assure you, I'll do everything I can to resolve 

this for you and avoid the possibility of this reoccurring.
co17	My goal was to provide you with an excellent customer service. Did I do that for you today? 
co18	Sorry for the trouble, we wouldn't let you down on our trust.
co19	Every single day we work towards one goal and that's 100% customer satisfaction.
co20	I got a quick fix for this and I'd recommend you do the same if you experience the same issue in future but I assure you that, it 

will fix the issue permanen
